tests/publiccloud/storage_perf.pm

```diff
@@ -67,11 +68,11 @@ sub analyze_previous_series {
 
 sub run {
     my ($self) = @_;
-    my $reg_code = get_var('SCC_REGCODE');
+
     my $runtime = get_var('PUBLIC_CLOUD_FIO_RUNTIME', 300);
     my $disk_size = get_var('PUBLIC_CLOUD_HDD2_SIZE');
     my $disk_type = get_var('PUBLIC_CLOUD_HDD2_TYPE');
-    my $url = get_var('PUBLIC_CLOUD_PERF_DB_URI');
+    my $url = get_required_var('PUBLIC_CLOUD_PERF_DB_URI');
```

Comment:
```suggestion
    my $url = get_var('PUBLIC_CLOUD_PERF_DB_URI');
```
This variable was so far optional and should remain that way.

Reply:
ok, previous form restored